accounts/tests.py

```diff
@@ -0,0 +1,33 @@
+from django.test import TestCase,Client
+from django.contrib.auth import get_user_model
+from .models import UserManager
+# Create your tests here.
+user = get_user_model()
+objects=UserManager
+class testMakeUserOk(TestCase):
```

Comment:
```suggestion
class CreateUserTests(TestCase):
```
クラス名の命名方法はPEP8に則りCapWords方式にしてください。また、テスト名は~Testsとしたほうが何をするテストなのか分かり易いです。クラス名は`CreateUserTests`として、その中に関数で`test_successful_create_user`や`test_invalid_create_user`などのユーザー作成時に関するテストを記述していってください。